src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Figure figure;
+
+    public Figure getRandomFigure() {
+        GeometricFigure geometricFigure = new GeometricFigureSupplier().getRandomFigure();
+        int random = new Random().nextInt(1, 100);
+        String colorSupplier = new ColorSupplier().getRandomColor().toLowerCase();
```

Comment:
The variable name 'colorSupplier' is misleading since it's actually storing a color string, not a supplier. Consider renaming it to 'color'.